cmd/tuple/read.go

```diff
@@ -70,7 +74,14 @@ func read(fgaClient client.SdkClient, user string, relation string, object strin
 		continuationToken = *response.ContinuationToken
 	}
 
-	return &openfga.ReadResponse{Tuples: &tuples}, nil
+	justKeys := make([]openfga.TupleKey, 0)
+	for _, tuple := range tuples {
```

Comment:
Instead of creating a new response, maybe we can just keep returning `*openfga.ReadResponse` as before, and on display just display `*reponse.Tuples` for the simple version

Reply:
Well, I thought on this, but then later I noticed that we don't have any logic inside of our commands. Normally we are externalizing it on a function. That's why I changed the type instead.

Reply:
Should I put the for on the command still? 😸

Reply:
Sorry for the late reply @gabrielbussolo!

Will merge this as is for now, and will do some cleanup before releasing (we're thinking we want to change the flag to something like `--simple` so that it can be used regardless of the output format).

But I don't want to delay this PR further as we discuss that.

Thanks for taking the initiative on this! 💪🏽 🎉